NewEvaluation.css

```diff
@@ -190,9 +204,22 @@ input[type=submit]{
 	float: right;
 	box-shadow: 5px 5px 5px #888888;
 }
-.spacing{
-	padding-top: 25vh;
-}
-.littleSpacing{
-	padding-top: 10vh;
+
+
+
+
+
+/*-----------------   Footer -----------------*/
+
+
+.BottomFooter{
+	bottom:0;
+    position:fixed;
+    z-index:150;
+    _position:absolute;
+    _top:expression(eval(document.documentElement.scrollTop+
```

Comment:
I, for one, am not terribly fond of CSS expressions, for multiple reasons:
* they violate separation of concerns
* they pollute the CSS with overly verbose computations
* they're non-standard CSS
* support has been [discontinued since IE8](https://msdn.microsoft.com/en-us/library/ms537634(v=vs.85).aspx)